dll/opengl/opengl32/wgl.c

```diff
@@ -744,7 +744,10 @@ BOOL WINAPI wglMakeCurrent(HDC hdc, HGLRC hglrc)
         /* Winetest conformance */
         if (GetObjectType( hdc ) != OBJ_DC && GetObjectType( hdc ) != OBJ_MEMDC)
         {
-            ERR( "Error: hdc is not a DC handle!\n");
+            if (hdc)
+            {
+                ERR("hdc (%p) is not a DC handle!\n", hdc);
```

Comment:
NB: If this remaining case is actually triggered, then we may want to log `GetObjectType(hdc)` result too.

Reply:
yes, I add this to the ERR macro.